### PR TITLE
Update OpenShift SDN documentation for multitenant plugin

### PR DESCRIPTION
**Pull request by dcbw**

### CODE REVIEW

**Comment by adellape** [comment]
@dcbw @rajatchopra My impression after finding https://github.com/openshift/origin/pull/3665 (unmerged atm) is that I probably shouldn't merge these docs here yet. Can you confirm? Also trying to gauge what OSE release this might be targeted at, so that I can get it into the OSE docs branch at the appropriate time.

cc @sdodson

**Comment by rajatchopra** [comment]
Yes, its fair to gate this PR with openshift/origin#3665

**Comment by dcbw** [comment]
Agreed, this should go in only after https://github.com/openshift/origin/pull/3665 does.

File: architecture/additional_concepts/sdn.adoc [hunk 1]
```diff
@@ -109,7 +111,7 @@ network footnote:[After this point, device names refer to devices on container
 B's host.] -> vxlan0 -> br0 -> vovsbr -> vlinuxbr -> lbr0 -> vethB -> eth0 (in
 B's netns)_
 
-== External Access to the Cluster Network
+==== External Access to the Cluster Network
```
Comment (adellape) [inline architecture/additional_concepts/sdn.adoc]: Is this only applicable to the ovssubnet plug-in, as its current placement suggests?

File: architecture/additional_concepts/sdn.adoc [hunk 2]
```diff
@@ -122,3 +124,97 @@ so that the master does not schedule containers on it.
...
+or receive packets from pods of a different project.
+
+However, the "default" project (which receives VNID 0) is more privileged in
+that it is allowed to communicate with all other pods, and all other pods can
+communicate with the "default" project.  This facilitates certain services
+(like the load balancer) that are run in containers themselves and must talk
+to all other containers.
+
+When OpenShift SDN is run on a node with the multitenant plugin, it configures
+five network devices:
```
Comment (adellape) [inline architecture/additional_concepts/sdn.adoc]: Five or six for this one?

**Comment by adellape** [comment]
@dcbw @rajatchopra Couple new comments inline. Also, I see https://github.com/openshift/origin/pull/3665 was merged the other day; is the content here still accurate?

File: architecture/additional_concepts/sdn.adoc (hunk 2 above)
Comment (rajatchopra) [inline architecture/additional_concepts/sdn.adoc]: Its five. vlinuxbr and vovsbr are a pair that are born together and die together.

File: architecture/additional_concepts/sdn.adoc (hunk 1 above)
Comment (rajatchopra) [inline architecture/additional_concepts/sdn.adoc]: Correct. Only for ovssubnet plugins.

**Comment by rajatchopra** [comment]
@adellape Nothing has changed since 3665 merged.

File: architecture/additional_concepts/sdn.adoc (hunk 2 above)
Comment (adellape) [inline architecture/additional_concepts/sdn.adoc]: OK thanks. Reason I ask is cuz it says five above for the ovssubnet list as well; should that be four then?

Comment (dcbw) [inline architecture/additional_concepts/sdn.adoc]: > OK thanks. Reason I ask is cuz it says five above for the ovssubnet list as well; should that be four then?

@adellape It should be four; fixed, rebased, and repushed.

Comment (adellape) [inline architecture/additional_concepts/sdn.adoc]: :+1:

**Pull request merged**